mdl/mlp_probe.py

```diff
@@ -29,7 +65,7 @@ def __init__(
             )
 
         output_dim = num_classes if num_classes > 2 else 1
-        sizes = [num_features] + [hidden_size] * (num_layers - 1)
+        sizes = [num_features] + [hidden_size] * (num_layers)
```

Comment:
It's unnatural to have num_layers - 1 hidden layers in an MLP. Defining linear probe separately so using num_layers hidden layers isn't an issue